packages/nova-lib/lib/server/meteor_patch.js

```diff
@@ -0,0 +1,98 @@
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+
+import { Utils } from '../modules/index.js';
+
+// our bindEnvironment keep the original function and auto rename the return function
+export const bindEnvironment = (fn) => {
```

Comment:
Thanks for the explanations, I get the point!! 👌  

Though when packages depend explicitly on `core` instead of `lib`, they cannot access it? 

> i know i wrote too many code on it.

Definitely not too many code dude! ✌️ _After writing thought: when I said minimize, it was about minimizing the impact of patching, not minimizing code_ 😄